prac_05/wimbledon.py

```diff
@@ -24,22 +24,3 @@ def process_data(data):
             champions[champion] = 1
         countries.add(country)
     return champions, sorted(countries)
-
-def display_champions(champions):
-    print("Wimbledon Champions:")
-    for champion, wins in champions.items():
```

Comment:
The dictionary containing champion names to their number of wins could be better renamed to 'champion_to_win_count' or 'champion_to_count'.